WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/utils/StatsDateFormatter.kt

```diff
@@ -100,7 +99,8 @@ class StatsDateFormatter
                 startCalendar.add(Calendar.DAY_OF_WEEK, -6)
                 return printWeek(startCalendar, endCalendar)
             }
-            MONTHS -> WordUtils.capitalize(outputMonthFormat.format(date))
+            MONTHS -> outputMonthFormat.format(date)
+                    .replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString() }
```

Comment:
Good question @oguzkocer ! 💯 

Actually, this conversion was done automagically, as a suggestion for the replacement of the deprecated `StringsJVM.capitalize(...)` function. If you click-enter on this deprecated method, the IDE will then suggest you this alternative. As such, this seems the safest choice to me, to trust Kotlin on that:

`String.JVM.kt` -> `L740` -> `@Deprecated`
```
/**
 * Returns a copy of this string having its first letter titlecased using the rules of the specified [locale],
 * or the original string if it's empty or already starts with a title case letter.
 *
 * The title case of a character is usually the same as its upper case with several exceptions.
 * The particular list of characters with the special title case form depends on the underlying platform.
 */
@Deprecated("Use replaceFirstChar instead.", ReplaceWith("replaceFirstChar { if (it.isLowerCase()) it.titlecase(locale) else it.toString() }"))
@DeprecatedSinceKotlin(warningSince = "1.5")
@SinceKotlin("1.4")
@WasExperimental(ExperimentalStdlibApi::class)
@kotlin.internal.LowPriorityInOverloadResolution // To avoid conflicts in function references, as this function was introduced later than common capitalize()
public fun String.capitalize(locale: Locale): String {
    if (isNotEmpty()) {
        val firstChar = this[0]
        if (firstChar.isLowerCase()) {
            return buildString {
                val titleChar = firstChar.titlecaseChar()
                if (titleChar != firstChar.uppercaseChar()) {
                    append(titleChar)
                } else {
                    append(this@capitalize.substring(0, 1).uppercase(locale))
                }
                append(this@capitalize.substring(1))
            }
        }
    }
    return this
}
```

Wdyt? 🤔

Reply:
That is .. _weird_, but I am sure there is a reason. (whether it's a good reason is another question 😓) I guess we leave it as is. 🤷‍♂️

Reply:
😄 I had the same exact thought, but ended up trusting JetBrains and Kotlin on the reason! 😄 

Btw, I can find the reason for you if you like, it is just that I never got down to it, maybe I am trusting JetBrains and Kotlin too much... 😝

Yea, let's leave it as is. 🤷

PS: Another reason I want to leave it as is, it is the fact that next time, when I go and do such replacement I want to have a point of reference that this was already done in the codebase, at least once, to give me that little more of a confidence that this is the correct thing to be doing. For example, see `StringExtensions.kt` and its `String.capitalizeWithLocaleWithoutLint(...)` extension function. In the past, I chose to just deprecate that, maybe next time I'll replace such deprecations instead. As such, when I wrote that code from scratch, I didn't want to use `capitalize(...)` and then suppress this deprecation, I said, let's do it right for new code... 🤞